### ISSUE TITLE
Installing 32 bit SQL Engine on 64 bit OS

### ISSUE THREAD

**Issue by GitEmde** [dsccommunity/SqlServerDsc#39]
Error:
Cannot find path 'HKLM:\SOFTWARE\Microsoft\Microsoft SQL Server\Instance Names\SQL' because it does not exist.  

Path should be: 
HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft SQL Server\Instance Names\SQL

**Comment by johlju** [comment]
Logs for this particular error has been submitted in issue #392.

**Comment by SphenicPaul** [comment]
Is it worth closing this? See [here](https://github.com/dsccommunity/SqlServerDsc/issues/392#issuecomment-744439413) in #392

**Comment by johlju** [comment]
I agree with the https://github.com/dsccommunity/SqlServerDsc/issues/392#issuecomment-744439413 so I'm closing this issue, but if anyone needs this support for 2012 R2 then I'm happy to review a PR for it.

**Comment by dan-hughes** [comment]
@johlju I think this can be closed too.

**Comment by johlju** [comment]
Yes, let us do that.
